src/form/Form.js

```diff
@@ -1,6 +1,6 @@
 import React, {useState} from 'react';
-import Input from "../inputComponent/Input";
-import SubmitButton from "../buttonComponent/SubmitButton";
+import Input from "../input/Input";
+import SubmitButton from "../button/SubmitButton";
 import './Foms.css'
```

Comment:
Please change component forder name from button to SubmitButton.

Same changes for Input. So "import Input from "../input/Input";" -> import Input from "../Input/Input";